### ISSUE TITLE
Proxy sometimes is not working in Windows

### ISSUE THREAD

**Issue by rodrigoeg** [ciandt-dev/Docker4Acquia#22]
Sometimes, when you execute "make windows" to start the containers, nginx proxy starts, but not properly configured.

Proxy needs to start before all other containers.

Possible solution currently being tested:
Change Makefile dependency order for proxy in Windows and Mac:

```
# run Docker4Acquia containers + Nginx proxy
windows: proxy run
mac: proxy run
```

**Comment by rodrigoeg** [comment]
Changes committed in the branch "issue/22". It still a draft of solution.
There are some pending items to be considered:

- [ ] As the order of makefile commands were changed, the messages are being printed in console in incorrect order. Maybe we will need to adde an additional instruction after run to print only the console messages. Example: `windows: proxy run print-message`
- [ ] It is not guaranteed that nginx proxy will be up and running with this change, but it reduces the change of this bug. An final solution would be waiting for nginx to start, and then start the other containers.

**Comment by lisotton** [comment]
I agree that output should be changed. Even with current order I think that messages are wrong. If you run `make windows` it will show a list for IPs to be included in `hosts` and then will show a new list. I think that the best option is to consolidate the output.
